packages/gatsby-plugin-typography/src/typography.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line no-undef
+module.exports = require(__TYPOGRAPHY_PLUGIN_CACHE_ENDPOINT__)
```

Comment:
Question here - is this because we can't use `__TYPOGRAPHY_PLUGIN_CACHE_ENDPOINT__` with ESM import syntax? So something like

`import typography from __TYPOGRAPHY_PLUGIN_CACHE_ENDPOINT__`

doesn't actually work?

If that's the case, maybe instead of using `definePlugin`, we could set webpack alias that would point to `path.join(cache.directory, `typography.js`)` and then we could have ES imports like that:
`import typography from "typography-plugin-cache-endpoint"` and alias would look like:

```js
setWebpackConfig({
    resolve: {
      alias: {
        "typography-plugin-cache-endpoint": path.join(cache.directory, `typography.js`),
      },
    },
  })
```

that would allow extra file that workaround ES import + definePlugin limitations?

Reply:
Sure, I think that would probably work.  Maybe something like
```js
export * from “typography-plugin-cache-endpoint”
```
The idea of that file is to just be a straight through proxy for the actual content in the user’s cache, so I think `export *` should work. 

This way would be better than relying on a magic constant, too.  Thanks!

 I’ll give it a try later and see how it turns out.

Reply:
Sure, I think that would probably work.  Maybe something like
```js
export * from “typography-plugin-cache-endpoint”
```
The idea of that file is to just be a straight through proxy for the actual content in the user’s cache, so I think `export *` should work. 

This way would be better than relying on a magic constant, too.  Thanks!

 I’ll give it a try later and see how it turns out.

Reply:
Okay, `export *` doesn't work, but has to be a default export anyway.  So `export { default } from ` works.

Check it out now.  I also updated the cache file that gets written out to use es6 import/export, too, instead of CommonJs